_extras/guide.md

```diff
@@ -27,13 +27,15 @@ for a look at how using checklists can save lives (and make many other things be
 
 > ## Setting Up [Trainer]
 >
+> 1.  Create an Etherpad (http://pad.software-carpentry.org/<nameofevent>
```

Comment:
There have been experiments with Google Docs instead, perhaps mention that as well?